### ISSUE TITLE
Phone Verification Issue

### ISSUE THREAD

**Issue by Computerfile** [ai-to-ai/Auto-Gmail-Creator#27]
Hi, I'm trying to use the auto-gmail-creator but Ive aquired the problem that the sms verification is a paid service and I would prefer to have the gmail creator work without having to pay for the sms verification. Do you mind getting in contact with me on discord to talk about the problem?

My discord is @graminoid Thank You!

**Comment by Finebody** [comment]
How much required to pay for sms verification? Any idea?

**Comment by xD-E-A-T-Hx** [comment]
Depends where you’re set for. Think it can be as low as 8 cent each. But go to the sms webpage it shows all the prices in it per one

**Comment by Computerfile** [comment]
Yeah, I didn't manage to find one though I guess I have to stick with the one you chose, I was kinda hoping to find a free alternative but it doesn't seem like theres any reliable ones out there :/ if you know one thatd be of great help

**Comment by Finebody** [comment]
> Yeah, I didn't manage to find one though I guess I have to stick with the one you chose, I was kinda hoping to find a free alternative but it doesn't seem like theres any reliable ones out there :/ if you know one thatd be of great help

Hey there! how did you get past the google error, "Sorry! We couldn't create your google account!" ? I am stuck at it. Anything helpful will be greatly appreciated.

**Comment by Computerfile** [comment]
Yeah I had that problem. If you're using selenium I recommend switching to undetected_chromedriver and changing the options to have a unique fingerprint every time (meaning randomizing a range of commonly used extensions, random battery levels, webgl rendering solutions window dimensions, user agents etc....) that will help to make it undetectable. I also read somewhere on an old stackoverflow thread that there is a common key used in the webdriver executable that may lead to detection (can't seem to find it right now though).

**Comment by xD-E-A-T-Hx** [comment]
yeah seleniumwire has javascript signatures in it which is detected by the google bots, best way atm is to modify the script to run on selenium and use chromedriver, you can edit the choredriver.exe looking for the cdc_ markers and replace them with something random like qwe_ as i have modified my script a lot, tbh im not sure which part has beaten the bot for now, my next obstacle is trying to send a cancel request to the sms api to cancel the number after its been used,

**Comment by Computerfile** [comment]
You could also make it that numbers get re used for emails (i.e one number can be used for 3-5 times) so that you don't have to reget a number everytime

**Comment by Lyenge** [comment]
Better use free bot from AutoPVA.com 
it worked perfectly fine for me.
